src/table-list-field/table-list-field.component.ts

```diff
@@ -63,6 +63,8 @@ export class TableListFieldComponent extends AbstractListFieldComponent implemen
 
   onFieldAdd(field: string) {
     this.keys.push(field);
+    setTimeout(() => {
```

Comment:
Because it is needed for ensuring that the new components that are about to be created where first added to the `elemTabIndexList`. But logically it doesn't make sense to move it to the `sortAndSynchronizeTabIndexes`.